tests/xz.rs

```diff
@@ -47,8 +36,8 @@ fn standard_files() {
 
 fn test_good(data: &[u8]) {
     let mut ret = Vec::new();
-    read::XzDecoder::new(data).read_to_end(&mut ret).unwrap();
```

Comment:
The 3 test cases that were excluded above need the multi-decoder to work.
Maybe it would be worth running `test_good` twice: once with `write::XzDecoder::new` (with the 3 uncovered test cases excluded) and once with `write::XzDecoder::new_multi_decoder`.